src/cube.js

```diff
@@ -168,27 +168,21 @@ class Cube {
     }
 
     renameMeasure(oldMeasureId, newMeasureId) {
-        if (oldMeasureId == newMeasureId) return this;
-
-        const cube = new Cube(this.dimensions);
-        Object.assign(cube.storedMeasures, this.storedMeasures);
-        Object.assign(cube.storedMeasuresRules, this.storedMeasuresRules);
-        Object.assign(cube.computedMeasures, this.computedMeasures);
-
-        if (cube.computedMeasures[oldMeasureId]) {
-            cube.computedMeasures[newMeasureId] = cube.computedMeasures[oldMeasureId];
-            delete cube.computedMeasures[oldMeasureId];
-        } else if (cube.storedMeasures[oldMeasureId]) {
-            cube.storedMeasures[newMeasureId] = cube.storedMeasures[oldMeasureId];
-            cube.storedMeasuresRules[newMeasureId] = cube.storedMeasuresRules[oldMeasureId];
-            delete cube.storedMeasures[oldMeasureId];
-            delete cube.storedMeasuresRules[oldMeasureId];
```

Comment:
Why we do not need anymore to `delete cube.storedMeasuresRules[oldMeasureId];` ?

Reply:
Good catch. Fixed now.